src/Controls/src/Core/Shell/ShellContent.cs

```diff
@@ -5,6 +5,7 @@
 using System.Collections.Specialized;
 using System.ComponentModel;
 using System.Reflection;
+using System.Security.Cryptography;
```

Comment:
Is this used anywhere?

Reply:
Removed, VSCode sometimes likes to add stuff :-(